### PR TITLE
Make timeout a param and increase default to 20 for docker_creds.py

### PR DESCRIPTION
**Pull request by themurph**
The new default method for verifying registry authentication has a hardcoded timeout of 10 seconds which is causing installations of 3.11 in some of our remote offices/datacenters to fail because the verify step takes 13-15 seconds. 

https://github.com/openshift/openshift-ansible/blob/b895add118ba4cd4d0899c364fce6a44d6239351/roles/lib_utils/library/docker_creds.py#L136-L138

This commit would make the timeout value be a module param as well as increase the default timeout to 20 seconds.

### CODE REVIEW

File: roles/lib_utils/library/docker_creds.py [hunk 1]
```diff
@@ -200,6 +206,7 @@ def run_module():
         password=dict(type='str', required=True, no_log=True),
         test_login=dict(type='bool', required=False, default=True),
         proxy_vars=dict(type='str', required=False, default=''),
+        test_timeout=dict(type='str', required=False, default='20'),
```
Comment (vrutkovs) [inline roles/lib_utils/library/docker_creds.py]:
```suggestion
        test_timeout=dict(type='int', required=False, default='20'),
```

Comment (themurph) [reply roles/lib_utils/library/docker_creds.py]: @mtnbikenc Thanks for catching that. Fixed

**Comment by sdodson** [comment]
GCP quota, other tests are passing.

**Pull request merged**